src/open_inwoner/utils/logentry.py

```diff
@@ -163,5 +168,6 @@ def system_action(
             "action_flag": LOG_ACTIONS[5],
             "message": message,
             "log_level": log_level,
-        },
+        }
+        | kwargs,
```

Comment:
Note the existing codebase doesn't really use overloaded operators so I'm not convinced we should sprinkle them around in random places. (eg: it is no longer simplistic code)